.gitignore

```diff
@@ -8,6 +8,8 @@
 
 # dependencies
 /node_modules
+/functions/node_modules
+/tools/screenshot-test/functions/node_modules
```

Comment:
If you remove the slash for the `/node_modules` above. Then it will include all `node_modules/` folders in the whole project.

Just do 

```
node_modules/
```